giotto/graphs/geodesic_distance.py

```diff
@@ -63,13 +67,15 @@ def _geodesic_distance(self, X):
     def fit(self, X, y=None):
         """Do nothing and return the estimator unchanged.
 
-        This method is there to implement the usual scikit-learn API and hence
+        This method is here to implement the usual scikit-learn API and hence
         work in pipelines.
 
         Parameters
         ----------
-        X : ndarray of sparse or dense arrays, shape (n_samples,)
+        X : ndarray of shape (n_samples,) or \
+            (n_samples, n_vertices, n_vertices)
             Input data, i.e. a collection of adjacency matrices of graphs.
+            Each adjacency matrix may be a dense or a sparse array.
```

Comment:
You could remove "a" from "or a" if desired. That is "dense or sparse array."

Reply:
Indeed. Done.